CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ if( CMAKE_SYSTEM_NAME MATCHES Linux )
 endif()
 
 option( ENABLE_ITT "Build targets with ITT instrumentation support (requires VTune)?" OFF )
+option( ENABLE_TOOLS "Build targets with additional tools" OFF )
```

Comment:
How about the code:
```
if (ENABLE_TOOLS)
  add_subdirectory(${CMAKE_HOME_DIRECTORY}/tools)
endif
```
right after line 66 in this file below. In that way we will avoid any changes and checks in the tools/* cmake files...